src/ucp/rndv/rndv.h

```diff
@@ -37,7 +37,7 @@ typedef struct {
     /* Size of the data for sending */
     size_t            size;
     /* RNDV proto opcode */
-    uint8_t           opcode;
+    int8_t            opcode;
```

Comment:
why needed?

Reply:
to fix Coverity warning about signed/unsigned comparison, because enum is converted to signed type (`UCP_RNDV_RTS_TAG_CANCELED` is a negative one)

Reply:
can we define UCP_RNDV_RTS_TAG_CANCELED as (uint8_t)UCS_ERR_CANCELED?

Reply:
yes, it is another option
done